src/QMCHamiltonians/tests/test_PairCorrEstimator.cpp

```diff
@@ -66,6 +66,8 @@ namespace qmcplusplus
 {
 TEST_CASE("Pair Correlation", "[hamiltonian]")
 {
+  outputManager.pause();
```

Comment:
This change adds difficulty for troubleshooting issues.
We probably need to extend the catch_main.cpp to allow passing an option to make the printout through if the default is off. It should not be hard coded.

If you just need to work on one test, pass the test name as an argument.

Probably just revert such changes to keep this PR moving.

Reply:
The tests spew so much unecessary information because of the need to setup objects.  If a test fails and if there is a need to debug remove this.  The default should not to print a huge amount of application output irrelevant to the test.  Also when running tests -V in terminal it really slows them down.  It's almost always the same output because we use the same input over and over for mocking.

Reply:
I will revert the changes in historical tests but I always include this in my tests as we are not testing the ability to blast stdout.

Reply:
That is why I put up #4090. I will put up a PR removing all the rest pause in tests after that one and this one get merged.